src/components/popup.jsx

```
@@ -1,13 +1,14 @@
 import React, { Component } from 'react';
+/* eslint react/prefer-stateless-function: 0 */
```

don't suppress this error

---

you can just make Popup a presentation component, checkout presentational v container components.

i.e.
```
const Component = props => (
 <render material goes here>
);


export default Component;
```